code/modules/mining/mine_items.dm

```diff
@@ -500,3 +500,72 @@
 	anchored = 1
 	layer = BELOW_MOB_LAYER
 	density = 0
+
+///Mining Base////
+
+/area/shuttle/auxillary_base
+	name = "Auxillary Base"
+
+
+/obj/machinery/computer/shuttle/auxillary_base
+	name = "auxillary base launch control"
+	icon = 'icons/obj/terminals.dmi'
+	icon_state = "dorm_available"
+	shuttleId = "colony_drop"
+	req_access = list(access_heads)
+	possible_destinations = null
+	clockwork = TRUE
+
+/obj/machinery/computer/shuttle/auxillary_base/Topic(href, href_list)
+	if(href_list["move"])
+		if(z != ZLEVEL_STATION)
+			usr << "<span class='warning'>You can't move the base again!</span>"
+			return 0
+	..()
+
+/obj/item/device/assault_pod/mining
+	name = "Landing Field Designator"
+	icon_state = "gangtool-purple"
+	item_state = "electronic"
+	icon = 'icons/obj/device.dmi'
+	desc = "Deploy to designate the landing zone of the auxillary base."
+	shuttle_id = "colony_drop"
+	dwidth = 11
+	dheight = 0
+	width = 23
+	height = 23
+	lz_dir = 1
+
+/obj/item/device/assault_pod/mining/attack_self(mob/living/user)
+	var/turf/T = get_turf(user)
+	if(T.z != ZLEVEL_MINING)
```

Comment:
You should totally add an admin override variable so players could trade their TC to drop a base on the station.